python/py-dm-tree/Portfile

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        deepmind tree 0.1.5
+name                py-dm-${github.project}
+revision 0
+
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         Deepmind tree is a library for working with nested \
+                    data structures.
+long_description    {*}${description} In a way, tree generalizes the builtin \
+                    map function which only supports flat sequences, and \
+                    allows to apply a function to each "leaf" preserving \
+                    the overall structure.
+
+checksums           rmd160  be34ef9fce02a8812f14b57a24f392649789399b \
+                    sha256  c8389c2e58cb02d4721985c6f003ab9b8f5cfe58e678b52501fc586cf9a0b724 \
+                    size    34622
+
+python.versions     37 38 39
+
+proc python_listify {tcl_list} {
+    set python_list {}
+    foreach item [lreverse ${tcl_list}] {
+        set python_list "'${item}', ${python_list}"
+    }
+    return ${python_list}
+}
+
+if {${name} ne ${subport}} {
+    PortGroup       bazel 1.0
+
+    bazel.version   3.7
+    bazel.clean_post_build \
+                    no
+    
+    # Disable bazel configure and build phase configuration
+    bazel.build_cmd
+    bazel.configure_cmd
+
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-six
+
+    depends_test-append \
+                    port:py${python.version}-absl \
+                    port:py${python.version}-attrs \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-wrapt
+
+    # diff -NaurdwB ./dm-tree-orig/setup.py ./dm-tree-new/setup.py | sed -E -e 's/\.\/dm-tree-(orig|new)*\/(setup\.py)(\.[[:alnum:]]+)*/\.\/setup.py/' > ./patch-setup.py.diff
+    patchfiles      patch-setup.py.diff
+
+    post-patch {
+        reinplace \
+            "s|@BAZEL_CMD@|[python_listify [concat {bazel} [option bazel.build_cmd_opts]]]|g" \
```

Comment:
@cjones051073 Note that the bazel PG does not allow access to `bazel_build_env`, which may be desired here.

Also, I'll follow up with a comment in #10390 that there's a good chance that these should be environment variables anyway, not shell variables, so that bazel's java subprocess has access to them.